tests/golos.charge_tests.cpp

```diff
@@ -35,15 +35,17 @@ class golos_charge_tester : public golos_tester {
         , charge({this, cfg::charge_name, _token_sym})
         , _users{N(alice), N(bob)}
     {
-        create_accounts({_code, cfg::token_name, cfg::charge_name, cfg::control_name, cfg::emission_name});
+        create_accounts({cfg::token_name, cfg::charge_name, cfg::control_name, cfg::emission_name});
         create_accounts(_users);
         produce_blocks(2);
-        install_contract(_code, contracts::vesting_wasm(), contracts::vesting_abi());
         install_contract(cfg::token_name, contracts::token_wasm(), contracts::token_abi());
         install_contract(cfg::charge_name, contracts::charge_wasm(), contracts::charge_abi());
     }
 
     void init(int64_t issuer_funds, int64_t user_vesting_funds) {
+        create_account(_code); 
+        install_contract(_code, contracts::vesting_wasm(), contracts::vesting_abi());
```

Comment:
1. please move this back to constructor. same in other tests where you moved it

Reply:
2. this line was initially wrong, the `_code` is current tester contract (contract, for which tests were created). so it should be `cfg::charge_name`. It's not your bug, but please fix `golos_charge_tester` constructor so it constructs `golos_tester(cfg::charge_name)` (this will make `_code` = `cfg::charge_name`) and restore if something broken